presto-main/src/test/java/com/facebook/presto/operator/scalar/TestUrlFunctions.java

```diff
@@ -49,6 +49,24 @@ public void testUrlExtractParameter()
         assertFunction("url_extract_parameter('foo', 'k1')", VARCHAR, null);
     }
 
+    @Test
+    public void testUrlEncode()
+    {
+        assertFunction("url_encode('http://test')", VARCHAR, "http%3A%2F%2Ftest");
+        assertFunction("url_encode('http://テスト')", VARCHAR, "http%3A%2F%2F%E3%83%86%E3%82%B9%E3%83%88");
```

Comment:
Don't put non-ascii characters in Java source code. Even though it's legal Java code, some tools don't handle them properly. If you're using IntelliJ, there's an intention to convert this into the corresponding \uxxxx encoding.